ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/access-lists.j2

```diff
@@ -1,5 +1,5 @@
 {# eos - extended access-lists #}
-{% if access_lists is defined and access_lists is not none %}
+{% if access_lists is arista.avd.defined %}
 !
 {%     for access_list in access_lists | arista.avd.natural_sort %}
 ip access-list {{ access_list }}
```

Comment:
missing check for `access_lists[access_list].sequence_numbers[sequence].action` in line 7